src/shell/shell_cmds.cpp

```diff
@@ -2242,3 +2243,290 @@ void DOS_Shell::CMD_VOL(char* args)
 	const auto trimmed_label = trim(label);
 	WriteOut(MSG_Get("SHELL_CMD_VOL_OUTPUT"), drive, trimmed_label, high, low);
 }
+
+static std::vector<std::string> cmd_move_parse_sources(const char*& args)
+{
+	bool parsing             = true;
+	bool inside_quote        = false;
+	bool stop_next_letter    = false;
+	bool continue_after_stop = false;
+	std::string word;
+	std::vector<std::string> sources;
+
+	// Sources are seperated by commas and terminated by a trailing space
+	while (parsing) {
+		switch (*args) {
```

Comment:
Would be more descriptive to set up a simple `ch` reference to args and use that in the parser.

Reply:
Sorry, I'm not sure what you mean by that exactly.

Reply:
I was just thinking `const auto curr_char = *args` or simply just `ch` is more descriptive. Then you can just advance the pointer with `++args` somewhere still.

Reply:
Got it.  Changed.